docker-compose.yml.mustache

```diff
@@ -20,7 +15,8 @@ auth:
   environment:
     - AUTHENTICATION_METHOD=fake
     - AUTHORIZATION_METHOD=fake
-    - FRONT_DOOR_HOST=http://$FRONT_DOOR_HOST:8080
+    - SESSION_HANDLER=redis
```

Comment:
Would really like to see these keys sorted — it was very useful for me when comparing configs.